pkg/virt-controller/watch/clone/clone_test.go

```diff
@@ -898,6 +898,28 @@ var _ = Describe("Clone", func() {
 			})
 		})
 
+		Context("Target VM name", func() {
```

Comment:
An idea for another test is: specify a specific name for the target, and verify it is assigned.
But we can also address this in a follow-up :)